libraries/chain/wasm_interface.cpp

```diff
@@ -727,13 +727,12 @@ class database_api : public context_aware_api {
       void db_idx64_remove( int iterator ) {
          return context.idx64.remove( iterator );
       }
-      int db_idx64_find_secondary( uint64_t code, uint64_t scope, uint64_t table, uint64_t& secondary, uint64_t& primary ) {
+      int db_idx64_find_secondary( uint64_t code, uint64_t scope, uint64_t table, const uint64_t& secondary, uint64_t& primary ) {
```

Comment:
Minor issue - here we are passing all read-only uint64_t parameters by value, except "secondary" which is passed by const reference. Seems inconsistent.